src/cares_wrap.cc

```diff
@@ -69,6 +69,19 @@ using v8::Value;
 
 namespace {
 
+inline uint16_t cares_get_16bit(const unsigned char* p) {
```

Comment:
@addaleax, but I think the code may be a bit more complicated.

befor:

```cpp
unsigned int ancount = cares_get_16bit(buf + 6);
```

after:

```cpp
unsigned int ancount = ntohs(*reinterpret_cast<int16_t*>(buf + 6));
```